plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtStartCommandWrapper.java

```diff
@@ -115,6 +117,16 @@ public Answer execute(final StartCommand command, final LibvirtComputingResource
                             break;
                         }
                     }
+
+                    try {
+                        File pemFile = new File(LibvirtComputingResource.SSHPRVKEYPATH);
+                        FileUtil.scpPatchFiles(controlIp, "/home/cloud", Integer.parseInt(LibvirtComputingResource.DEFAULTDOMRSSHPORT), pemFile, LibvirtComputingResource.systemVmPatchFiles, LibvirtComputingResource.BASEPATH);
```

Comment:
same as above - (we shouldn't assume that cloud user exists, or use /tmp or privileged /root/patch/...)